modules/migration-terminology.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+[id="migration-terminology_{context}"]
+= {mtc-short} terminology
+
+[cols="1,3a", options="header"]
+.{mtc-short} terminology
+|===
+|Term |Definition
+|Source cluster |Cluster from which the applications are migrated.
+|Destination cluster^[1]^ |Cluster to which the applications are migrated.
+|Replication repository |Object storage used for copying images, volumes, and Kubernetes objects during indirect migration or for Kubernetes objects during direct volume migration or direct image migration.
+
+The replication repository must be accessible to all clusters.
```

Comment:
I think the description should be `The replication repository must be accessible 'from' all clusters.`

Reply:
Just had another look at your comment. I think "to" is the correct word in this context. It does not imply a direction (replication repository -> clusters). "accessible to" means that the replication repository can be accessed by all clusters. However, I might rewrite this.